README.md

```diff
@@ -143,6 +144,51 @@ outlined in the [Building images on non SLE distributions](#building-images-on-n
 section.
 
 
+### Building images on-demand SLE instances in the public cloud
+
+When building container images on SLE instance that where launched as so-called
+"on-demand" or "pay as you go" instances on a Public Cloud (as AWS, GCE or Azure)
+some additional steps have to be performed.
+
+For installing packages and updates the "on-demand" public cloud instance are
+connected to a public cloud specific update infrastructure which is based around
+some RMT servers operated by SUSE on the various Public Cloud Providers.
+
+To be able access this update infrastructure the instances need to perform some
+additional steps to locate the required service and authenticate with them.
+(Some more details on this are outline in a
+[Blog Series on suse.com starting here](https://suse.com/c/a-new-update-infrastructure-for-the-public-cloud/).
+
+In order to be able to build containers on this type of instances a new service
```

Comment:
`In order to build containers`

I make the same "mistake" too :)

Reply:
:heavy_check_mark: